modules/javafx.controls/src/test/java/test/com/sun/javafx/scene/control/infrastructure/ControlBehaviorTestExample.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package test.com.sun.javafx.scene.control.infrastructure;
+
+import java.lang.ref.WeakReference;
+import java.util.Collection;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import com.sun.javafx.scene.control.behavior.BehaviorBase;
+
+import static org.junit.Assert.*;
+import static test.com.sun.javafx.scene.control.infrastructure.ControlSkinFactory.*;
+
+import javafx.scene.control.Control;
+
+/**
+ * Example of writing a test for a streak of similar issues,
+ * here f.i. memory leak in behavior.
+ *
+ * Note: doesn't have postfix _Test_ to not be included in
+ * default testing.
+ */
+@RunWith(Parameterized.class)
+public class ControlBehaviorTestExample {
```

Comment:
This test does not execute when run as part of :controls:test. We need to rename this class (and file) name to end with 'Test'. I tried doing it and lot of failures are seen.

Reply:
that's the point - they are examples of how-to use it and see our tasks :)

Will address your other comments in a minute - and add two real tests (on skin/behavior memory leaks with the misbehaving classes removed: then we'll have both the passing classes and informal ignores on what's going wrong.

Reply:
added SkinMemoryLeakTest/BehaviorMemoryLeakTest as (more or less ;) outlined in the bug report: in the parameter list we start with all controls/classes and then remove the failing explicitly. Future fixes of the leaking classes will have to remove the fixed from the excluded. 

Hmm ..  now probably should remove the xxTestExamples (there existence doesn't seem to have been clear enough anyway ;)

Reply:
Yes. It is better to remove this xxTestExamples file.